README.md

```diff
@@ -45,6 +45,8 @@ override:
 
 ```ruby
 class NameOfTask < Schlepper::Task
+  USE_TRANSACTION = true
```

Comment:
Ah for backwards compatibility you'll need to add `controls_transaction?` to Schlepper::Task though.